tests/test_request.py

```diff
@@ -51,12 +50,11 @@ def setup_method(self):
         }
         self.id_token = encode_token(self.cognito_user)
         self.resp = Request(
-            headers=CIMultiDict({"Content-Type": "application/json"}),
-            uri_params={},
-            # pylint issue #214
             method="GET",
+            uri_params={},
             body="",
             context={},
+            headers=CIMultiDict({"Content-Type": "application/json"}),  # pylint issue #214
```

Comment:
It looks like with the new version of `pylint`, [the configuration](http://pylint.pycqa.org/en/latest/technical_reference/features.html#similarities-checker) started to work correctly. That means we can increase the number of lines taken for similarity comparison:
```cfg
# setup.cfg

[pylint.SIMILARITIES]
min-similarity-lines = 9
```
I don't want to say we should set `9` but it is the minimal number with which we don't get an error. Also, in the future, we will have to do something with that anyway 😉

Reply:
I wouldn't put that number at all - this is valid thing for even 1 line, but in this case does not make sense.

Reply:
Yep, `pylint` messed up with those similarities 😕  Let's keep workarounds.